ocp_resources/datavolume.py

```diff
@@ -106,40 +107,47 @@ def __init__(
         self.secret = secret
         self.content_type = content_type
         self.size = size
-        self.access_modes = access_modes
+        self.access_modes = (
+            access_modes if use_storage_api or access_modes else self.AccessMode.ROX
+        )
```

Comment:
``` self.access_modes = (
            access_modes if use_storage_api or access_modes else self.AccessMode.ROX
       )
```
Why this is needed? You still use access_mode in both cases.

Reply:
will update so the DV won't have a default value